website/docs/workloads/ctstraffic/ctstraffic.md

```diff
@@ -0,0 +1,24 @@
+# CtsTraffic
+ctsTraffic is a highly scalable client/server networking tool giving detailed performance and reliability analytics
+
+* [Official CtsTraffic Documentation](https://github.com/microsoft/ctsTraffic/tree/master/documents)
+* [Official Release](https://github.com/microsoft/ctsTraffic/tree/master/Releases/2.0.3.0)
+
+## What is Being Measured?
+ctsTraffic is designed as a classic client/server model, where the server will wait listening for client requests, while the client initiates connections to servers. ctsTraffic servers will accept any number of connections from any number of clients; ctsTraffic clients can target and make connections to one or more servers.
```

Comment:
how is this different from the ntttcp tests? for bandwidth as example, which one should we use?

Reply:
Target of NTTTCP/Latte and ctstraffic is different. Former ones are more inclined towards performance where as ctstraffic is more for reliability. It is a simple benchmark which is highly scalable with a lot of options to control time, number of clients, number of servers, exact buffer and total data etc. that can be run in earlier phases of HW qualification/scenarios where we are more concerned about reliability of networking among several machines, we dont care how they performed, we just care that they should be able to communicate and able to send and receive data.